library/cwm/src/lib/cwm/service_widget.rb

```diff
@@ -30,6 +30,10 @@ def initialize(service)
       self.handle_all_events = true
     end
 
+    def default_action=(value)
+      @service_widget.default_action=(value)
```

Comment:
This looks weird. Likely a copy and paste leftover.

Reply:
+1

Reply:
when I see so much redirection, maybe we should consider using Forwardable here.

Reply:
I already have it in the changes I'm preparing